scheduler/src/main/resources/org/pentaho/platform/scheduler2/messsages/messages.properties

```diff
@@ -19,6 +19,7 @@ QuartzScheduler.ERROR_0005_FAILED_TO_RESUME_JOB=Quartz failed to resume job.
 QuartzScheduler.ERROR_0006_FAILED_TO_GET_SCHEDULER_STATUS=Unable to get Quartz scheduler status.
 QuartzScheduler.ERROR_0007_FAILED_TO_GET_JOB=Failed to get job "{0}"
 QuartzScheduler.ERROR_0008_SCHEDULING_IS_NOT_ALLOWED=You don't have scheduling permissions for this file.\nContact your administrator for assistance.
+QuartzScheduler.ERROR_0009_SCHEDULING_IS_NOT_ALLOWED_AFTER_CHANGE=Due to user/role changes job "{0}" is not able to be executed by "{1}"
```

Comment:
No
Digging into the issue  - was found that is either not possible or hardly implement to give alert for admin about changing permissions or roles (that due to that changes he would break some scheduled jobs).

So just as fix was decided to not run in case of lack  of permissions and provide noticeable debug warn info for admin in logs. The message wasn`t approved by UX - just thought it could be changed in any time - especially during localization process.